### PR TITLE
Upgrade for fjord activation

### LINKED ISSUE #11: Upgrade for fjord activation
### Description
As Lisk Mainnet and Sepolia testnet nodes are upgraded for Fjord activation, it is necessary to update the information to run Lisk nodes.

### Acceptance Criteria

- Set `--override.fjord` for both op-node and op-geth/op-reth
- Node should run without any issues

### Additional Information
https://docs.optimism.io/builders/node-operators/network-upgrades#activations

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #11 

### How was it solved?

- [x] Set `--override.fjord` for both op-node and op-geth/op-reth
- [x] Node should run without any issues
- [x] Update reth dockerfile (Clone the one forked by base-org to support fjord hardfork)
- [x] Fix `fork-sync.yml` file

### How was it tested?

- [x] Local

### CODE REVIEW

File: geth/geth-entrypoint [hunk 1]
```diff
@@ -46,6 +46,10 @@ if [ "${OP_NODE_OVERRIDE_CANYON+x}" = x ]; then
     ADDITIONAL_ARGS="$ADDITIONAL_ARGS --override.canyon=$OP_NODE_OVERRIDE_CANYON"
 fi
 
+if [ "${OVERRIDE_FJORD+x}" = x ]; then
+    ADDITIONAL_ARGS="$ADDITIONAL_ARGS --override.fjord=$OVERRIDE_FJORD"
+fi
```
Comment (sameersubudhi) [inline geth/geth-entrypoint]: I think, this shouldn't be optional. We should exit when the `OVERRIDE_FJORD` env variable isn't supplied.

The check can be like:
```
if [[ -z "OVERRIDE_FJORD" ]]; then
    echo "expected OVERRIDE_FJORD to be set" 1>&2
    exit 2
fi
```
and the start command:
```
# Start service
./geth \
    --datadir="$GETH_DATA_DIR" \
    --verbosity="$VERBOSITY" \
    --http \
    --http.corsdomain="*" \
    --http.vhosts="*" \
    --http.addr=0.0.0.0 \
    --http.port="$RPC_PORT" \
    --http.api=web3,debug,eth,net,engine \
    --authrpc.addr=0.0.0.0 \
    --authrpc.port="$AUTHRPC_PORT" \
    --authrpc.vhosts="*" \
    --authrpc.jwtsecret="$OP_NODE_L2_ENGINE_AUTH" \
    --ws \
    --ws.addr=0.0.0.0 \
    --ws.port="$WS_PORT" \
    --ws.origins="*" \
    --ws.api=debug,eth,net,engine \
    --metrics \
    --metrics.addr=0.0.0.0 \
    --metrics.port="$METRICS_PORT" \
    --syncmode="$OP_GETH_SYNCMODE" \
    --gcmode="$OP_GETH_GCMODE" \
    --maxpeers=100 \
    --rollup.sequencerhttp="$SEQUENCER_HTTP" \
    --rollup.halt=major \
    # --op-network="$OP_NODE_NETWORK" \
    --port="$P2P_PORT" \
    --rollup.disabletxpoolgossip=true \
    --override.fjord=$OVERRIDE_FJORD \
    $ADDITIONAL_ARGS # intentionally unquoted
```

File: op-node-entrypoint [hunk 2]
```diff
@@ -44,8 +46,13 @@ export OP_NODE_P2P_ADVERTISE_IP=$PUBLIC_IP
 
 echo "$OP_NODE_L2_ENGINE_AUTH_RAW" > "$OP_NODE_L2_ENGINE_AUTH"
 
+if [ "${OVERRIDE_FJORD+x}" = x ]; then
+    ADDITIONAL_ARGS="$ADDITIONAL_ARGS --override.fjord=$OVERRIDE_FJORD"
+fi
+
```
Comment (sameersubudhi) [inline op-node-entrypoint]: Same as above.

File: reth/reth-entrypoint [hunk 3]
```diff
@@ -6,6 +6,7 @@ RPC_PORT="${RPC_PORT:-8545}"
 WS_PORT="${WS_PORT:-8546}"
 AUTHRPC_PORT="${AUTHRPC_PORT:-8551}"
 METRICS_PORT="${METRICS_PORT:-6060}"
+ADDITIONAL_ARGS=""
```
Comment (sameersubudhi) [inline reth/reth-entrypoint:9]: I couldn't find the support for the `--override.fjord` flag on RETH. Let's dig deep further on this.

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

File: reth/reth-entrypoint (hunk 3 above)
Comment (nagdahimanshu) [reply reth/reth-entrypoint:9]: I checked and found out that these property is already set in `reth`. 
Ref PR: https://github.com/paradigmxyz/reth/pull/8645

Comment (sameersubudhi) [reply reth/reth-entrypoint:9]: Then, we'll need to update the reth version, right? It's not supported on reth v1.0.0 that we've set in the `reth/Dockerfile`.

File: .env.sepolia [hunk 4]
```diff
@@ -2,6 +2,7 @@ GENESIS_FILE_PATH=sepolia/genesis.json
 SEQUENCER_HTTP=https://rpc.sepolia-api.lisk.com
 OP_NODE_OVERRIDE_CANYON=0
 RETH_CHAIN=lisk-sepolia
+OVERRIDE_FJORD=1716998400
```
Comment (shuse2) [inline .env.sepolia:5]: where did this number come from?

**Review by shuse2 — COMMENTED** [review]
Other than comment from @sameersubudhi LGTM

File: .env.sepolia (hunk 4 above)
Comment (sameersubudhi) [reply .env.sepolia:5]: https://docs.optimism.io/builders/node-operators/network-upgrades#activations

**Review by sameersubudhi — APPROVED** [review]
LGTM and works as expected against the mainnet 👍

**Review by shuse2 — APPROVED** [review]

**Pull request merged**